1-hello-world/app.yaml

```diff
@@ -12,5 +12,5 @@
 # limitations under the License.
 #
 # [START runtime]
-runtime: nodejs8
+runtime: nodejs10
```

Comment:
Just making sure with @steren that y'all want these using the latest version :)

Reply:
I actually made the request to change the samples.
Node 10 is what we want to direct users towards by default, since it is the new LTS.